controllers/suite_test.go

```diff
@@ -91,13 +90,18 @@ func setup() error {
 	// the purpose of these tests
 	envy.Set("POD_NAMESPACE", "azureoperator-system")
 
+	// Force the secret naming version to 2 for these tests
+	// Note: This can be removed to run the tests in the old v1 naming scheme (they
+	// should still pass)
+	envy.Set("AZURE_SECRET_NAMING_VERSION", "2")
```

Comment:
Wondering if I should make the default v2 and then force people to set it back to v1. That's what I did in the Helm chart but I didn't do it in code... should I do it in both?

Reply:
Yes, for consistency.